### PR TITLE
Update go-dockerclient dependency

### PR DESCRIPTION
**Pull request by juanrhenals**
r? @aaithal @samuelkarp @richardpen

### CODE REVIEW

File: agent/Godeps/Godeps.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
 {
 	"ImportPath": "github.com/aws/amazon-ecs-agent/agent",
 	"GoVersion": "go1.6",
-	"GodepVersion": "v60",
-	"Packages": [
-		"./..."
```
Comment (samuelkarp) [inline agent/Godeps/Godeps.json]: Looks like you might have only saved one package instead of all the packages used by the agent.

Comment (juanrhenals) [inline agent/Godeps/Godeps.json]: Fixed. Just updated the commit.

**Comment by samuelkarp** [comment]
I'm seeing a test failure:

```
--- FAIL: TestStartTimeoutThenStart (0.00s)
        controller.go:158: missing call(s) to *engine.MockDockerClient.StopContainer(is equal to containerId)
        controller.go:165: aborting test due to missing call(s)
```

I don't think this is related to your change since I'm also seeing it without your change, but we need to get this fixed.

**Comment by samuelkarp** [comment]
:+1: since I think the test failure is unrelated

**Pull request merged**